corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html

```diff
@@ -13,11 +13,18 @@
         }
     </style>
 {% endblock %}
+{% block js %}{{ block.super }}
```

Comment:
cool, I think we can put on the back burner for now but might be worth doing at some point